### DISCUSSION TITLE
GR-Procedure-PlaceOfPerformance non-repeatable (SDK 1.12)

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#1001] [Q&A] [answered]
This version 12 eliminates the repeatable from this group, this is because it refers to a non-repeatable ND, but its parent ND is. As shown below in the fields.json:

![image](https://github.com/user-attachments/assets/7ad913c9-55fc-4ab8-b6cf-9d6a0c600e12)

No sé si es un error del SDK que no lo han tenido en cuenta, o de la forma de recorrer el modelo para llegar al modelo físico.

Thanks in advance,
Alex

**Comment by YvesJo** [comment, marked as answer]
Hi,
There may be multiple locations involved in the procurement, however for a given location, there may only be one address.
